### ISSUE TITLE
Negative racetime.gg scores

### ISSUE THREAD

**Issue by Nitro6003** [racetimeGG/racetime-app#138]
Hello so this is my issue with racetime.GG that you cannot find out how negative your score is and I know that a negative number could mean that some people could get discouraged but I think if you want to see it I think that should be allowed I also think it would be acceptable to show whether you get plus or minus points regardless of your if you’re negative so for example if you’re -1000 and you do get a plus number it should show that in the last race that you played that if you earn any points it would be plus X if you earned less points or Any more negative number then that would be acceptable to so that’s what I would like to say

**Comment by scaramangado** [comment]
Maybe it would be possible to just cut off at 0 points artificially.

**Comment by Nitro6003** [comment]
> Maybe it would be possible to just cut off at 0 points artificially.

Even putting a zero woud be ok but how do we know when we start gaining points?  I think there should a way to see it all the time.  I really want to know when I gain or loose points to see what is happening

**Comment by scaramangado** [comment]
I mean actually behave as if users have 0 points. Like moving everyone who is negative atm to 0 and prevent anyone from falling below 0.

That or just showing all scores, whether they look flattering or not.

**Comment by Nitro6003** [comment]
> I mean actually behave as if users have 0 points. Like moving everyone who is negative atm to 0 and prevent anyone from falling below 0.
> 
> That or just showing all scores, whether they look flattering or not.

Yes and we can fall back to zero but in races where we gain points we always see it. I’m on discord if you want to discuss further.
